docs/content/glossary/glossary.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: Glossary
+---
+
+# Flow Glossary
+
+## All words and definitions related to Blockchain and Flow
+
+## **A**
+
+## Authentication
+[Authentication](https://docs.onflow.org/fcl/reference/authentication/#gatsby-focus-wrapper) connects users to their wallets, 
+and the wallet provides the account information and services through FCL.
+
+## Authorization
+Authorization is a process by which a server determines if the client has permission to use a resource or access a file. In FCL, 
+An authorization service knows the Account and the Key to sign the transaction when the service is sent to FCL during authentication.
+
+## Authorization Envelope
+The transaction authorization envelope contains both the [transaction payload](https://docs.onflow.org/concepts/transaction-signing/#payload) and the payload signatures.
+
+## Authorizers
+[Authorizers](https://docs.onflow.org/flow-go-sdk/#transactions) are accounts that authorize a transaction to read and mutate their 
+resources. A transaction can specify zero or more authorizers, depending on how many accounts the transaction needs to access.
+
+## Account
+An [account](https://docs.onflow.org/flow-token/concepts/#segmented-transaction-fees) on Flow is a record in the chain state that holds 
+the address, balance, public keys, code, and storage.
+
+## Account Balance
+In Flow, the [account balance](https://docs.onflow.org/concepts/accounts-and-keys/#accounts) is the number of tokens available in your 
+wallet. 
+
+## Account storage
+The area of the [account](https://docs.onflow.org/cadence/language/accounts/#account-storage) used to store resource assets
+
+## Address
+A blockchain address is like a bank account for crypto assets. Every address is unique, and they are 26-35 alphanumeric characters in length. 
+They are generated from the private key, which is required to send or receive data from one address to another.
+
+## Access node
+[Access nodes](https://docs.onflow.org/node-operation/node-roles/#access) act as a proxy to the Flow network. The Access node routes 
+transactions to the correct Collection node and routes state queries to available nodes (while likely caching state to answer queries 
+on time in the future).
+
+## **B**
+
+## Block
+Blocks are data structures within the blockchain database, where transaction data in a cryptocurrency blockchain are permanently recorded.
+
+## Blocto
+[Blocto](https://portto.com/) is a new smart contract wallet meant to address long-standing issues with private key management, 
```

Comment:
remove `new smart contract`. it is a wallet that can handle authentication and authorization

also replace `gas fees` with `transaction fees`. I don't think we call them gas fees on Flow

Reply:
fixed